### ISSUE TITLE
Upgrade works and books page to v2 of template

### ISSUE THREAD

**Issue by jdlrobson** [internetarchive/openlibrary#1208]
We will upgrade all our major pages to v2 of the template.

Upgrading has various benefits
- serves non-render blocking JavaScript
- more modern version of jQuery
- more mobile friendly
- potential for slimmer render blocking CSS

The home page has currently been migrated.

This task tracks migrating work and book pages
https://openlibrary.org/works/OL17903635W/Vortigern's_Garden
https://openlibrary.org/books/OL11088089M/Between_Golden_Jaws_(Hallowmere)

A page can be considered migrated when:
- #test-body-mobile is present in the HTML
-  the page is bug free.
- renders on a mobile device without horizontal scrolling

**Comment by jdlrobson** [comment]
Blocked on #1206

**Comment by jdlrobson** [comment]
Superseded by #1249
